rllib/algorithms/ars/ars_tf_policy.py

```diff
@@ -86,10 +86,17 @@ def __init__(self, obs_space, action_space, config):
             for _, variable in self.variables.variables.items()
         )
 
-    def compute_actions(self, observation, add_noise=False, update=True, **kwargs):
+    def compute_actions(self, obs_batch=None, add_noise=False, update=True, **kwargs):
```

Comment:
This unifies the signature with the rest of RLlib so we can make proper use of it with kwargs